EventFilter/CTPPSRawToDigi/plugins/SealModule.cc

```diff
@@ -0,0 +1,9 @@
+#include "FWCore/PluginManager/interface/ModuleDef.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "EventFilter/CTPPSRawToDigi/plugins/CTPPSPixelDigiToRaw.h"
+#include "EventFilter/CTPPSRawToDigi/plugins/CTPPSTotemDigiToRaw.h"
+
+DEFINE_FWK_MODULE(CTPPSPixelDigiToRaw);
+DEFINE_FWK_MODULE(CTPPSTotemDigiToRaw);
```

Comment:
SealModule.cc is not really needed.
The ```DEFINE_FWK_MODULE``` can be in the corresponding .cc file where the producers are defined